managed-ledger/src/main/java/org/apache/bookkeeper/mledger/ManagedCursorMXBean.java

```diff
@@ -70,4 +70,33 @@
      */
     long getPersistZookeeperErrors();
 
+    /**
+     * Add write data to a ledger of a cursor (in bytes).
+     *
+     * @param size Size of data written to cursor (in bytes)
+     */
+    void addWriteCursorLedgerSize(long size);
```

Comment:
`long size` mark as `final` maybe better.

Reply:
Yes, I have tried, but the checkstyle said that 'Redundant 'final' modifer '.

Reply:
ok, you can declare `final` in the method implementation in `ManagedCursorMXBeanImpl.java`

Reply:
Ok, done